examples/client/Locomotion/src/Components/CardRow/index.tsx

```diff
@@ -49,7 +51,7 @@ const margin = `margin-${Start()}`;
 const TextContainer = styled(View)`
   justify-content: center;
   ${margin}: 16px;
-  width: 80%;
+  width: ${({ noSvg } : { noSvg: boolean}) => (noSvg ? '95%' : '80%')};
```

Comment:
Use flex

Reply:
undid the changes, it actually doesnt affect anything